### PR TITLE
Add the onOpen, onClose and onChalExpired callbacks

### LINKED ISSUE #111: Add missing callbacks
We want to add the missing callbacks to the React wrapper, see [documentation](https://docs.hcaptcha.com/configuration).
- open-callback
- close-callback
- chalexpired-callback

Expected Example:
```js
<HCaptcha 
  onOpen={handleOpen}
  onClose={handleClose}
  onChalExpired={handleChallengeExpired}
/>
```

### PR DESCRIPTION
**Pull request by zoryana94**
close #111

Знімок екрана 2022-01-21 о 13 44 50
Знімок екрана 2022-01-21 о 13 15 42

### CODE REVIEW

**Comment by alidzm** [comment]
Missing props declaration for the component. Let's add https://github.com/hCaptcha/react-hcaptcha/blob/master/types/index.d.ts

**Review by wontem — CHANGES_REQUESTED** [review]
The code looks good. Please also update `README.md` with the new callbacks

File: src/index.js [hunk 1]
```diff
@@ -202,6 +208,24 @@ class HCaptcha extends React.Component {
...
+        this.props.onClose();
+      }
+    }
+
+    handleChallengeExpired () {
+      if (this.props.onChalExpired) {
+        this.props.onChalExpired();
+      }
+    }
+
```
Comment (wontem) [inline src/index.js:228]: Please also consider using `if (!isApiReady || isRemoved) return` in these callbacks.

**Review by wontem — CHANGES_REQUESTED** [review]

**Review by wontem — COMMENTED** [review]
Looks good. @brdlyptrs please take a quick look since you probably have a better understanding of how these callbacks should work or maybe you aware of some edge cases

File: src/index.js [hunk 2]
```diff
@@ -39,12 +39,16 @@ class HCaptcha extends React.Component {
       this.renderCaptcha = this.renderCaptcha.bind(this);
       this.resetCaptcha  = this.resetCaptcha.bind(this);
       this.removeCaptcha = this.removeCaptcha.bind(this);
+      this.isCaptchaReady = this.isCaptchaReady.bind(this);
```
Comment (brdlyptrs) [inline src/index.js:42]: Can this be `isReady`?

**Comment by brdlyptrs** [comment]
Last thing, can you bump the version, may be `1.1.0`?

**Review by brdlyptrs — APPROVED** [review]

**Review by wontem — APPROVED** [review]

**Pull request merged**